src/Util/AdminObjectAclData.php

```diff
@@ -145,7 +147,7 @@ public function getAclRoles()
      *
      * @return AdminObjectAclData
      */
-    public function setAcl(Acl $acl)
+    public function setAcl(MutableAclInterface $acl)
```

Comment:
Changing this param signature is a BC-break, isn't it ?

Reply:
@VincentLanglet no, since `Acl` implements `MutableAclInterface`